Apps/Database/src/Models/EmailFormatCode.cs

```diff
@@ -13,28 +13,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //-------------------------------------------------------------------------
-namespace HealthGateway.Database.Context
+namespace HealthGateway.Database.Models
 {
-    using HealthGateway.Database.Models;
-    using Microsoft.EntityFrameworkCore;
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.ComponentModel.DataAnnotations.Schema;
+    using HealthGateway.Database.Constant;
 
-    /// <summary>
-    /// The database context used by the web client application.
-    /// </summary>
-    public class WebClientDbContext : BaseDbContext
+    #pragma warning disable CS1591 // self explanatory simple model
+    #pragma warning disable SA1600 // self explanatory simple model
+    public class EmailFormatCode : AuditableEntity
     {
-        /// <summary>
-        /// Initializes a new instance of the <see cref="WebClientDbContext"/> class.
-        /// </summary>
-        /// <param name="options">The DB Context options.</param>
-        public WebClientDbContext(DbContextOptions<WebClientDbContext> options)
-            : base(options)
-        {
-        }
+        [Key]
+        [Column("EmailFormatCodeId")]
+        public Guid Id { get; set; }
 
-        /// <summary>
-        /// Gets or sets the set of Audit Events.
-        /// </summary>
-        public DbSet<UserProfile> UserProfile { get; set; }
+        [Required]
+        [MaxLength(4)]
+        public string FormatCode { get; set; }
```

Comment:
same as above, docs missing

Reply:
warnings disabled again due to simple model.